ios/RNTPTDocumentView.m

```diff
@@ -1317,6 +1317,36 @@ - (void)importAnnotations:(NSString *)xfdfString
     if (!updateSuccess || error) {
         @throw [NSException exceptionWithName:NSGenericException reason:error.localizedFailureReason userInfo:error.userInfo];
     }
+    
+    return [self getAnnotationFromXFDF:xfdfString];
+
+}
+
+-(NSMutableArray<NSDictionary *> *)getAnnotationFromXFDF:(NSString *)xfdfString
+{
+    NSMutableArray<NSDictionary *> *annotations = [[NSMutableArray alloc] init];
+    PTFDFDoc *fdfDoc = [PTFDFDoc CreateFromXFDF:xfdfString];
+    PTObj *fdf = [fdfDoc GetFDF];
+    if (fdf != nil) {
```

Comment:
The Obj-C PDFNet wrapper always returns a value, and you have to check the `IsValid` method instead:
```suggestion
    if ([fdf IsValid]) {
```